api/types.proto

```diff
@@ -884,3 +884,39 @@ message MaybeEncryptedRecord {
 	bytes data = 2;
 	bytes nonce = 3;
 }
+
+// SecurityProfiles specifies per-architecture security configuration/permissions
+message SecurityProfiles {
+	// Privileges required to run in Windows
+	oneof windows {
+		WindowsRawSecurityProfile windows_raw = 1;
+	}
+	// Privileges required to run in Linux
+	oneof linux {
+		LinuxRawSecurityProfile linux_raw = 2;
+	}
+}
+
+// WindowsRawSecurityProfile provides any low level privileges a windows container needs
+message WindowsRawSecurityProfile {
+	// JSON string containing credential specs created with gMSA to use with managed service accounts in Windows
+	// (see https://blogs.msdn.microsoft.com/containerstuff/2017/01/30/create-a-container-with-active-directory-support/)
+	string credential_spec = 1;
+}
+
+
+// LinuxRawSecurityProfile provides any low level privileges a linux container needs
+message LinuxRawSecurityProfile {
+	message Device {
+		string path = 1;
+	}
+
+	// Linux capabilities required by the container
+	repeated string capabilities = 1;
+
+	// Devices needed by the service
+	repeated Device devices = 2;
```

Comment:
Good point - I left it in because I wasn't sure if we'd need anything else.  https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md#devices has a bunch of other options, but https://docs.docker.com/engine/extend/config/ only has the name, description, and path.  (I figured the name and description are probably not necessary, but wasn't sure)